src/data/EpiPoint.ts

```diff
@@ -1,7 +1,10 @@
 import type EpiDate from './EpiDate';
 
 export default class EpiPoint {
-  constructor(private readonly date: EpiDate, private readonly value: number) {}
+  constructor(
+    private readonly date: EpiDate,
+    private readonly value: number,
+  ) {}
```

Comment:
this looks like just whitespace changes...  if so, we should exclude this file from the PR.